.storybook/pages/CoursePlannerEdit/CoursePlannerEdit.tsx

```diff
@@ -499,7 +508,7 @@ export const CoursePlannerEdit = () => {
               // above it only when the flex-direction is column, so
               // this utility class was written with a media query
               // that matches the behavior of .page-header
-              <ButtonGroup className="u-margin-top-xl-mobile">
+              <ButtonGroup className={buttonClassName}>
```

Comment:
You can just use the your imported class name here directly and skip the variable.
```suggestion
              <ButtonGroup className={styles['button--mobile']}>
```